airbyte-webapp/.eslintrc

```diff
@@ -19,7 +19,7 @@
   "rules": {
     "jsx-a11y/label-has-associated-control": "error",
     "curly": "warn",
-    "css-modules/no-undef-class": ["error", { "camelCase": true }],
+    "css-modules/no-undef-class": "off",
```

Comment:
Just curious, why disable the whole rule?

Reply:
https://airbytehq-team.slack.com/archives/C039SG3DQ8K/p1662658406699359

we actually want to remove the whole plugin

Reply:
@tealjulia, and I didn't remove the whole plugin, because we are still using rule `css-modules/no-unused-class` (it's from the same plugin)

Reply:
> @tealjulia, and I didn't remove the whole plugin, because we are still using rule `css-modules/no-unused-class` (it's from the same plugin)

Hmm... And how do we use it with `"off"` flag? Just curious 😄

Reply:
@matter-q we want to remove the entire plugin.  But I am good with that being a separate issue/PR to keep this one from getting cluttered.

Reply:
> > @tealjulia, and I didn't remove the whole plugin, because we are still using rule `css-modules/no-unused-class` (it's from the same plugin)
> 
> Hmm... And how do we use it with `"off"` flag? Just curious 😄

We have such rules 😏

- "css-modules/no-undef-class": "off",
- "css-modules/no-unused-class": ["error", { "camelCase": true }],

And I'm talking about the second one 😃

Reply:
> > > @tealjulia, and I didn't remove the whole plugin, because we are still using rule `css-modules/no-unused-class` (it's from the same plugin)
> > 
> > 
> > Hmm... And how do we use it with `"off"` flag? Just curious 😄
> 
> We have such rules 😏
> 
> * "css-modules/no-undef-class": "off",
> * "css-modules/no-unused-class": ["error", { "camelCase": true }],
> 
> And I'm talking about the second one 😃

Oh, gotcha gotcha 👍